home/app_manager.py

```diff
@@ -173,14 +185,17 @@ def __init__(self, app, with_version_selector=False):
             (self.version_selector.installed_version, "value"),
             transform=self._formatted_version,
         )
+        ipw.dlink(
+            (self.version_selector.version_to_install, "value"),
+            (self.app, "version_to_install"),
```

Comment:
Why is this needed?

Reply:
Because the dataclass `_AiiDALabApp` is for store the information of the installed app and it is "Static" in terms of App it self. It has no class variable `version_to_install` in this base dataclass. From my understanding, the `version_to_install` is the information given and changed by the widget which wraps the `AiiDALabApp` traitlets. So here I using `dlink` to make `version_to_install` of `self.app` changed passively depend on the `self.version_selector.version_to_install`.